src/main/java/it/gov/pagopa/spontaneouspayment/exception/AppError.java

```diff
@@ -8,8 +8,10 @@
 public enum AppError {
 
     ORGANIZATION_NOT_FOUND(HttpStatus.NOT_FOUND, "Organization not found", "Not found the Organization Fiscal Code %s"),
+    ORGANIZATION_NOT_ENABLED(HttpStatus.UNPROCESSABLE_ENTITY, "Organization not enabled", "The organization is not in suitable state [Organization Fiscal Code = %s; Status = %s]"),
```

Comment:
Status = %s it's not useful, we already have "Organization not enabled"

Reply:
Right reporting. Corrected